Renamed the folder and committed changes for upcoming Release 55.1 (#1830)
EnterpriseDB/docs: product_docs/docs/migration_toolkit/55/06_building_toolkit.properties_file.mdx

-   `SRC_DB_PASSWORD` must contain the password of the specified user.

!!! Note
    - Make sure to append the optional parameter `?tinyInt1isBit=false` in the MySQL Connector/J JDBC Driver URL if datatype `tinyInt(1)` is used to store byte values other than 0 and 1 in the MySQL source database.    

>     - Due to a bug in the MySQL Connector/J JDBC Driver 8.0.26, the migration of foreign key constraints fails in certain cases. So it is recommended not to use this driver for migrating data using Migration Toolkit. You may use MySQL Connector/J JDBC Driver 8.0.25 driver instead.

Reviewer: ```suggestion
    - Due to a bug in the MySQL Connector/J JDBC Driver 8.0.26, the migration of foreign key constraints fails in certain cases. EDB recommends not to use this driver for migrating data using Migration Toolkit. You may use MySQL Connector/J JDBC Driver 8.0.25 driver instead.
```
